### PR TITLE
Add description related to compaction rate limit

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add note about the side effect of auto-tuned limiter and how to turn it off.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4497
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@taboki PTAL, thanks

File: tikv-control.md [hunk 1]
```diff
@@ -432,6 +432,28 @@ tikv-ctl --host ip:port modify-tikv-config -n raftstore.sync-log -v false
 success
 ```
 
+如果 compaction rate 的速率限制导致待 compaction 字节数积压，可以禁用 `rate-limiter-auto-tuned` 配置项或调高 compaction 相关的流量限制。示例如下：
```
Comment (tabokie) [inline tikv-control.md:435]:
```suggestion
如果 compaction 的流量控制导致待 compact 数据量 (compaction pending bytes) 堆积，可以禁用 `rate-limiter-auto-tuned` 配置项或调高 compaction 相关的流量阈值。示例如下：
```

**Review by tabokie — APPROVED** [review]
LG

**Pull request merged**